### ISSUE TITLE
Fire OS 5.2.7.4

### ISSUE THREAD

**Issue by umonkey13** [codefaktor/FTVLaunchX#12]
Seem to also be having issues since this update boot to launcher works but home button doesn't! Anyone else have this issue??

**Comment by 6uhrmittag** [comment]
I just tried FTVLaunchX and it looks like I have them same issue: booted right into the selected app, but no button works.
Also running 5.2.7.4

@umonkey13 My original FireTV remote app(on android phone) seems to work fine. Tab on the home button brings me right back to the original FireTV Dashboard. 

Edit: Sorry, my issue was probably related to the App "AirScreen" that I was testing in parallel. It now works again.
AirScreen slowed the stick down to the point of freezing. I unplugged the Stick 2 times and it lost the connection to the remote and even wifi completely.  After rebooting the 3. or 4. time, I re-inserted the battery into the remote and it magically re-connected to the stick. After uninstalling AirScreen everything works fine now. (I also get the known issue #7 )

**Comment by ghost** [comment]
Can confirm that my Fire Stick on 5.2.7.4 does not launch FTVLaunchX using home button but does launch on startup.

**Comment by ploquets** [comment]
Yes, I can confirm that as well. Only Home button is not working anymore.

**Comment by notNSANE** [comment]
any updates for 5.2.7.6? (current version)

**Comment by ErikAbele** [comment]
Yes, the issues are known and I probably won't be able to find time to fix this anytime soon... I'm sorry.

**Comment by loststone2** [comment]
I'm having the same issue

**Comment by AndreStork** [comment]
I found that Amazon blocked every accessibility service expect for VoiceView, causing home button to redirect into KFTV Launcher. Maybe disabling a service may work but I don't really know much about it
